CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ IF(NOT CLIENT_ONLY)
 
   SET(BOOST_COMPONENTS system thread regex)
   SEARCH_FOR_BOOST ()
-  FINDPYTHON(2.7 EXACT)
+  FINDPYTHON(3 REQUIRED)
```

Comment:
This will break python 2. Did you tried with `FINDPYTHON()` ?

Reply:
I have both python 2 and python 3 installed. What about having a separate branch of gepetto-viewer-corba for python 3? Or allowing the user to choose the python version with a CMake variable?

Reply:
If you use only `FINDPYTHON()`, you can already choose the python version you want at compilation time with CMake variables. 

For example:
```
-DPYTHON_EXECUTABLE=${PYTHON}
-DPYTHON_INCLUDE_DIR=${PYTHON_INCLUDE}
-DPYTHON_LIBRARY=${PYTHON_LIB}
```

I am not sure that you need all of those, the first might be enough.

Reply:
That is a bit tedious to type. I would prefer a CMake variable like `PYTHON_VERSION`.

Reply:
for reference: https://github.com/nim65s/robotpkg/blob/master/mk/sysdep/python.mk#L90

Reply:
@jmirabel yes, that would be better, but for this I suggest to open an issue / PR in the cmake submodule

Reply:
So I tried with just `-DPYTHON_EXECUTABLE=/usr/bin/python3.5` on 16.04, and it worked:
```
-- Found PythonInterp: /usr/bin/python3.5 (found version "3.5.2") 
-- PythonInterp: /usr/bin/python3.5
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython3.5m.so (found version "3.5.2") 
-- PythonLibraries: /usr/lib/x86_64-linux-gnu/libpython3.5m.so
-- PythonLibraryDirs: /usr/lib/x86_64-linux-gnu
-- PythonLibVersionString: 3.5.2
```

Reply:
I am fine with `PYTHON_EXECUTABLE` if we document it somewhere.

Reply:
we also have to remove `FINDPYTHON` in `src/CMakeLists.txt`

Reply:
I tried `-DPYTHON_EXECUTABLE=/usr/bin/python3.5` and I get:
```
-- Found PythonInterp: /usr/bin/python3 (found version "3.5.2") 
-- PythonInterp: /usr/bin/python3
-- Found PythonLibs: /usr/lib/x86_64-linux-gnu/libpython2.7.so (found version "2.7.12") 
-- PythonLibraries: /usr/lib/x86_64-linux-gnu/libpython2.7.so
-- PythonLibraryDirs: /usr/lib/x86_64-linux-gnu
-- PythonLibVersionString: 2.7.12
```
`libpython3.5` is installed on my computer but not `libpython3.5-dev`. Anyway, I should get an error and I don't.

Reply:
@jmirabel can you try with https://github.com/nim65s/jrl-cmakemodules/commit/bed90c47ef3fe868f17e1d8f01384da117c1ebe5 ? :)